Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -72,16 +72,16 @@ data DConfirmReq = DConfirmReq
   { bookingId :: Id DRB.Booking,
     customerMobileCountryCode :: Text,
     customerPhoneNumber :: Text,
-    fromAddress :: DBL.LocationAddress,
-    toAddress :: DBL.LocationAddress,
+    fromAddress :: DL.LocationAddress,
+    toAddress :: DL.LocationAddress,
```

Comment:
will last request only come as we are passing destination id and in address we are taking from this req